src/SFA.DAS.Apprenticeships.Api.Client/ITrainingProgrammeExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using SFA.DAS.Apprenticeships.Api.Types;
+
+namespace SFA.DAS.Apprenticeships.Api.Client
+{
+    public static class ITrainingProgrammeExtensions
+    {
+        public static bool IsActiveOn(this ITrainingProgramme course, DateTime date)
+        {
+            return GetStatusOn(course.EffectiveFrom, course.EffectiveTo, date) == TrainingProgrammeStatus.Active;
+        }
+
+        public static TrainingProgrammeStatus GetStatusOn(this ITrainingProgramme course, DateTime date)
+        {
+            return GetStatusOn(course.EffectiveFrom, course.EffectiveTo, date);
+        }
+
+        public static int FundingCapOn(this ITrainingProgramme course, DateTime date)
+        {
+            //todo: would probably be better to return int? null or throw if out of range
```

Comment:
I realised you may have coped this in from elsewhere? but we probably should resolve and remove the todo